experiments/transfer_learn_benchmark.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+# Train PPO2 experts using reward models from experiments/imit_benchmark.sh
+
+TIMESTAMP=$(date --iso-8601=seconds)
+REWARD_MODELS_DIR=reward_models
+LOG_ROOT="output/train_experts/${TIMESTAMP}"
+RESULTS_FILE="results.txt"
+ALGORITHM="gail"
+extra_configs=""
+
+
+SEEDS="0 1 2"
+ENVS="cartpole mountain_car acrobot half_cheetah "
```

Comment:
Why is this a different set of environments to `train_experts.sh`? (I'm guessing this is a GAIL vs AIRL thing?)

Reply:
Good catch, I'm missing the custom AIRL envs. I think I removed them previously for debugging purposes

Reply:
Hmmm actually I think what happened is that this is meant to be exactly the same as the environments used in `imit_benchmark_config.csv`. Which only has GAIL environments right now.

Reply:
I made `train_experts` and `imit_benchmark` have the same environments (superset of GAIL and AIRL environments).

`imit_benchmark_config.csv`, which is used to generate the reward models needed by this transfer learning script, originally had GAIL environments only. This was because I was confused about some missing experiment specifications from the AIRL paper (number of demonstrations used in particular I think).

To add the swimmer, maze, and custom/disabled ant environments into the imit_benchmark CSV, I just winged it and chose some arbitrary settings. Thinking that I can be more careful about these settings later.

Reply:
Seems like a good solution!